source/2015-06-12-panter-die-demokratische-firma.html.markdown

```diff
@@ -0,0 +1,76 @@
+
+Operative Entscheide werden einmal wöchentlich an einem Meeting behandelt. Das
+Meeting ist nicht obligatorisch, allerdings kann es nicht als Arbeitszeit
+verrechnet werden. Jeder Mitarbeiter kann teilnehmen und bei jedem Entscheid ein
+Veto einlegen, wenn er mit einem Antrag nicht einverstanden ist.
+
+Um Themen wie Wachstum, Mitbestimmung und Marktpositionierung zu behandeln,
+werden Workshops durchgeführt, an denen alle Mitarbeitenden teilnehmen.
+
+## Die Vorteile und warum es funktioniert
+
+Wichtige Faktoren für den Erfolg sind die Selektion der Mitarbeitenden und die
+Transparenz bei der Kommunikation. Bewerber werden bei uns nicht nur zu einem
+Vorstellungsgespräch eingeladen, sondern in einer weiteren Runde auch zu einem
+Apéro, einem Weihnachtsessen oder einem anderen Anlass. Ziel dabei ist, dass
+möglichst viele Mitarbeitende den Kandidaten/die Kandidatin kennenlernen. Wenn
+alle Mitarbeitenden damit einerstanden sind, wird die Person eingestellt. Somit
+wird die Wahrscheinlichkeit, dass neue Mitarbeitende zu unserer Kultur passen
+massiv erhöht. Für den Bewerber/die Bewerberin wiederum hat dieses Vorgehen den
+Vorteil, dass auch er/sie weiss, was ihn/sie bei Panter erwartet. Beide Seiten
+profitieren von der transparenten Kommunikation.
+
+Durch die Einbindung der Mitarbeitenden in operative Entscheide werden diese von
+einer breiten Basis getragen und kontroverse Themen können diskutiert werden.
+Dies hat zur Folge, dass das Verständnis für die Entscheide gefördert wird,
+selbst wenn jemand nicht zu 100% mit dem gefällten Entscheid übereinstimmt.
+
+Da alle Mitarbeitenden am Erfolg/Misserfolg beteiligt sind, macht es nur Sinn,
+dass sie auch die Unternehmensentwicklung beeinflussen können. Dies Erhöht die
+Wahrscheinlichkeit, dass die Unternehmung generisch wächst und die
+Weiterentwicklung im Sinne aller Mitarbeitenden geschieht.
+
+## Die Herausforderungen
+
+Damit dieses System funktioniert, müssen die Mitarbeitenden auch bereit sein,
+die an sie abgegebene Verantwortung auch wahrzunehmen. Je nach Persönlichkeit
+kann dies eine Belastung darstellen.  Falls Mitarbeiter nur am Lohn und nicht am
+Erfolg der Unternehmung interessiert sind, würde das System wahrscheinlich nicht
+funktionieren. Aus diesem Grund ist die Selektion der Mitarbeitenden besonders
+wichtig.  Die Beteiligung am Erfolg bedeutet auch, dass Risiken von allen
+Mitarbeitenden getragen werden. Ein Lohn der von Monat zu Monat unterschiedlich
+ist, ist nicht jedermans Sache.
+
+## Fazit
+
+Durch die flache Hierarchie und die Beteiligung aller Mitarbeitenden bei
+operativen und strategischen Entscheiden wird die Zufriedenheiden und die
+Identifikation der Mitarbeitenden mit der Unternehmung erhöht.  Von den
+Mitarbeitenden fordert dies die Bereitschaft sich an unternehmerischen
+Entscheiden zu beteiligen. Ausserdem müssen sie sich in einem dynamischen Umfeld
+mit laufend neuen Herausforderungen wohlfühlen.  Wir würden uns freuen, wenn
```

Comment:
Den letzten Satz würde ich weglassen, diese "wir würden uns freuen"-Sätze finde ich etwas aussagelos... Ich würde eher einen auf die Firma Panter bezogenen Satz anfügen, sowas in der Richtung, dass wir die Herausforderung transparent anpacken werden oder sowas.